cv32e40s/tests/programs/custom/interrupt_test/interrupt_test.c

```diff
@@ -201,7 +201,7 @@ __attribute__((interrupt ("machine"))) void u_sw_direct_irq_handler(void)  {
 	    "j m_fast13_irq_handler\n"
 	    "j m_fast14_irq_handler\n"
 	    "j m_fast15_irq_handler\n"
-        ".option pop\n"
+      ".option pop\n"
```

Comment:
There seems to be a mix of tabs and spaces in this file as well.

![image](https://user-images.githubusercontent.com/75431602/223393476-a6c1aeca-2055-42dd-b6da-eec7683681cd.png)

Reply:
With this one I have a feeling that the indentation is intended, but the mix of spaces and tabs should be fixed.